src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -77,14 +85,14 @@
 
 
         GenomeLocSortedSet includeSortedSet;
-        if (intervalStrings.isEmpty()){
+        if (getIntervalStrings().isEmpty()){
             // if -L argument isn't given but -XL is, set include to all of the territory
```

Comment:
We checked `intervalsSpecified()`, so we know either -L or -XL was specified, by checking -L here we now know that -XL must have been specified.  I've updated the comment to make it clearer.